console/src/app/modules/nav/nav.component.html

```diff
@@ -183,7 +172,7 @@
                     [routerLinkActive]="['active']"
                     [routerLinkActiveOptions]="{ exact: false }"
                     [routerLink]="['/org-settings']"
-                    *ngIf="['policy.read'] | hasRole | async"
```

Comment:
might be an edge case, but what if there were two orgs with custom policies... if i remove one of them, i'm not able to customize or even reset the policies of the remaining org

Reply:
thats true.. its just the navigation which is remove.. the page is still available as `/org-settings` though

Reply:
@hifabienne do you have an opinion on this? ☝️ 

since it's an edge case i guess it's ok like that

Reply:
just had a discussion with @hifabienne ... we think we'll just go with that

but how does the user know that he needs to go to `Instance` to set the settings in case the link to org settings in not visible?
What's your opinion on changing the `Instance` button to `Default Settings`?

And please check if any images in docs need to be updated

Reply:
we could improve that with additional milestone queries. `Default Settings` is not suitable for a button in that position. Better would be `Settings` and then make more prominent on the page that those settings are the fallback for any created org 🤔

Reply:
name of button is also changed to default settings